Documentation/FileFormats.md

```diff
@@ -0,0 +1,58 @@
+# Serenity Formats
+
+This document summarizes all non-standard file formats, data formats, and mime-types used by Serenity.
+
+## glyph/x-fonteditor (Clipboard-only)
+
+Requires the metadata-fields, `char` (usually encoded as a string of length 1: the codepoint as a character),
+`width`, and `height` (encoded as a decimal number).
+
+The data is encoded in width times height many bytes, either 0 (clear) or 1 (set).
+
+## image/x-serenityos (Clipboard-only)
+
+Requires the metadata-fields `width`, `height`, `scale`, `format` (see `Gfx::BitmapFormat`), and `pitch`, encoded as decimal strings.
+
+The data is encoded according to `Gfx::determine_storage_format(BitmapFormat)`, so either as
+BGRx8888, BGRA8888, RGBA8888, or 8-bit pallette index. Note that the pallette is not transferred.
+
+## text/uri-list (Clipboard-only)
+
+Newline-delimited set of URIs.
+
+## Application File (`*.af` files)
+
+These files are human-readable and are a subset of the INI-format, have no easily detectable filemagic, and define System Menu entries.
```

Comment:
```suggestion
These files are human-readable and are a subset of the INI-format, have no easily detectable filemagic, and define System Menu entries.

They are stored in [`/res/apps`](../Base/res/apps).
```

Reply:
Eh, alright, I'll add it in, but I would normally assume the user is able to run `git ls-files *.af` by themselves.

Reply:
Sure, everyone can do that. But you forget that this file is also viewable online, and hyperlinks are a nice thing :^)
I'd prefer a simple link over navigating there using the GitHub UI.